README/README.zh-CN.md

```diff
@@ -1,5 +1,5 @@
 <h1>
-  <img src="Images/p2mmbanner_logo.png" alt="p2mmbanner_logo" width="500" height="300" align="left">
+  <img src="../Images/p2mmbanner_logo.png" alt="p2mmbanner_logo" width="500" height="300" align="left">
```

Comment:
May I ask what this `../` is for for the path to the logo?

Reply:
Why is this needed exactly?

Reply:
Because i storage the file in ”Readme“ folder. need `../` to call the logo.
or just storage in root will better?

Reply:
Normally it already targets the logo just fine without `../`, normally it starts at root and looks for it from there. But I didn't know this also worked.